tools/esp8266/html/setup.html

```diff
@@ -102,7 +102,7 @@ <h1>Setup</h1>
                     <fieldset>
                     <legend class="des">MQTT</legend>
                         <label for="mqttAddr">Broker / Server IP</label>
-                        <input type="text" class="text" name="mqttAddr" value="{MQTT_ADDR}"/>
+                        <input type="text" class="text" name="mqttAddr" value="{MQTT_ADDR}" maxlength="32" />
```

Comment:
the maxlengh should come from define and not hard coded

Reply:
Thx for review

Reply:
Won't fix that for now, bc maybe it will be solved by reworked html/webserver things.